cfgov/jinja2/v1/your-story/index.html

```diff
@@ -122,7 +122,7 @@ <h3>Have a money question?</h3>
                                 <a class="a-link
                                           a-link__jump
                                           a-link__icon-after-text"
-                                   href="/askcfpb">
+                                   href="/ask-cfpb">
```

Comment:
Might as well add the trailing slash to avoid a redirect?

```suggestion
                                   href="/ask-cfpb/">
```